metricpublisher/schemas/log_event.json

```diff
@@ -0,0 +1,129 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "Schema for the log_event lambda request object",
+    "type": "object",
+    "properties": {
+        "request_id": {
+            "type": "string"
+        },
+        "metric_data": {
```

Comment:
For the attributes that have properties under it, have definition files for them separately and then ref them. See example https://github.com/awslabs/aws-serverless-dynamodb-event-sourcing/tree/master/app/schemas